### PR TITLE
Info for peerDependencies #patch

### LINKED ISSUE #197: 'react-animate-height' not found
Not sure if a bug or not, but figured I'd post this. 

After adding ui-component to a new project, when I import any component (not just an accordion) I receive this error: 

```
Could not find dependency: 'react-animate-height' relative to '/node_modules/@sendgrid/ui-components/accordion/accordion-panel.js'
```

When I add that library, I get the following: 

```
Could not find dependency: 'lodash' relative to '/node_modules/@sendgrid/ui-components/filterable-list.js'
```

Adding lodash makes it work. 

---

Here's an example: https://codesandbox.io/s/1ym1kvv0y3

### PR DESCRIPTION
**Pull request by reedsa**
Closes #197 

## Merge Checklist

- [x] Code reviewed by a frontend lead (Steve Kinney, Alex Wicks, Chris Chalstrom, Alex Thomsen)
- [ ] Workflow: Title has [semver](http://semver.org/) bump level defined (#major, #minor, #patch, or #noversion)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,5 +109,15 @@
...
+  },
+  "peerDependencies": {
+    "lodash": "^4.17.4",
+    "node-sass": "^4.8.3",
+    "raf": "^3.4.0",
+    "react": "^16.4.0",
+    "react-animate-height": "^2.0.1",
+    "react-color": "^2.14.1",
+    "react-dom": "^16.4.0",
+    "react-select": "^2.0.0"
```
Comment (Ditofry) [inline package.json]: do we still need these in the `dependency` block or can we axe em?

File: package.json [hunk 2]
```diff
@@ -36,8 +36,6 @@
   "author": "SendGrid <careers@sendgrid.com>",
   "license": "MIT",
   "dependencies": {
-    "classnames": "^2.2.5",
```
Comment (Ditofry) [inline package.json]: any particular reason `className` was removed?

File: package.json (hunk 1 above)
Comment (reedsa) [reply package.json]: I wasn't able to remove them from dependencies, not sure if it's something wrong with how yarn installs peers and how this package is configured but from a Yarn article:

> https://yarnpkg.com/blog/2018/04/18/dependencies-done-right/ 
> The peerDependencies object guarantees that, for each entry, any package that requires you will be requested to provide a copy of the dependency listed here, ideally matching the version you requested. It also guarantees that you’ll be able to access this dependency through require(), and finally it also guarantees that the return of require() will be exactly the same version & instance than the one your parent would obtain if they were to require() it themselves.

I would have thought yarn installs them for the project on `yarn install` and also warns consumers of ui-components that they must also install the dependencies if not installed.

Would definitely prefer them to be removed as dependencies!

File: package.json (hunk 2 above)
Comment (reedsa) [reply package.json]: It appears that we're doing our own thing with classnames and not using this dependency. Correct me if I'm wrong, maybe it's used and I missed it.

Comment (Ditofry) [reply package.json]: no that makes sense.  I was just being lazy and asking here instead of digging 👍

File: package.json (hunk 1 above)
Comment (Ditofry) [reply package.json]: Interesting.  Sounds good to me, thanks for the link 💯

File: README.md [hunk 3]
```diff
@@ -5,8 +5,16 @@ Re-useable React components based on the SendGrid's Style Guide.
...
+
+Install devDependencies with:
+```
+yarn add node-sass -D
+```
+
+Install dependencies with:
 ```
-npm install @sendgrid/ui-components
+yarn add lodash raf react react-animate-height react-color react-dom react-select @sendgrid/ui-components
```
Comment (chrischalstrom) [inline README.md]: i'm a little confused, shouldn't all these other deps like lodash be installed automatically local to the project when i add ui-components?

**Pull request merged**